public/cartridge.css

```diff
@@ -0,0 +1,215 @@
+/* grabbed from https://replit.com/@MaxWofford/cartridge-experiment#cartridge/style.css */
```

Comment:
Could this be rewritten to use CSS Modules (add be put in `components`) or inline styles? I'd like to help sort the repo's files